coderd/tracing/exporter.go

```diff
@@ -45,6 +48,17 @@ func TracerProvider(ctx context.Context, service string, opts TracerOpts) (*sdkt
 		closers = []func(context.Context) error{}
 	)
 
+	if opts.DataDog {
+		// See more:
+		// https://docs.datadoghq.com/tracing/metrics/runtime_metrics/go/
+		dd := ddotel.NewTracerProvider()
+		closers = append(closers, func(_ context.Context) error {
+			// For some reason, this doesn't appear to actually wind down
+			// the goroutines.
+			return dd.Shutdown()
+		})
```

Comment:
Can't you just append the `dd.Shutdown` function instead of wrapping it in a closure?

Reply:
Ah nvm, disregard.

Reply:
It doesn't match the signature.